common/test/acceptance/tests/studio/test_import_export.py

```diff
@@ -12,6 +12,25 @@
 
 
 # pylint: disable=no-member
+class TemplateTestMixin(object):
```

Comment:
Seems like a lot of work to add a one-line test to 4 test classes! Nevertheless it does the trick I suppose... I'd be interested to hear other peoples opinions... my first reaction was that this 'hides' the test and might cause difficulty when trying to understand what's going on when the test fails.